R/mrmash_wrapper.R

```diff
@@ -230,13 +222,7 @@ compute_coefficients_univ_glmnet <- function(X, Y, alpha, standardize, nthreads,
   r <- ncol(Y)
 
   linreg <- function(i, X, Y, alpha, standardize, nthreads, Xnew) {
-    if (nthreads > 1) {
-      registerDoFuture()
-      plan(multicore, workers = nthreads)
-      paral <- TRUE
-    } else {
-      paral <- FALSE
-    }
+    paral <- FALSE
```

Comment:
same thing here -- let's remove this parameter. Also, renove the `nthreads` parameter in this context.